### ISSUE TITLE
[ADE 508 Audit]: 508 Audit Report for 44578 MyHealTheVet (MHV) on VA.GOV: 508 Audit Report Findings Support - Defect 7 | Critical

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#113315]
> [!NOTE]
> **QUICK SUMMARY:** _Child ticket of https://github.com/department-of-veterans-affairs/va.gov-team/issues/113312 .._

### Defect 6 | Critical | Created:
_WCAG 1.1.1 Non-text Content | All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below._

Throughout the product, decorative images are read by assistive technology. For example, on the Medications details | Veterans Affairs screen (ALBUTEROL SULFATE 2MG TAB), under Your refill request for this medication is taking longer than expected, “Completed Graphic” is read. (New: 06/27/2025)

Defect 7 | Critical | Created:
WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

On the Medications | Veterans Affairs screen, under Medications list, the grouping of information is conveyed visually but cannot be understood by assistive technology—the medications and the associated information is read as unstructured text. (New: 06/27/2025)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by coforma-jamie** [comment]
[Comment in parent ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/113312#issuecomment-3028801163)
